src/commands/coverage/upload.ts

```diff
@@ -38,14 +37,35 @@ import {
   renderSuccessfulUploadCommand,
   renderUpload,
 } from './renderer'
-import {detectFormat, isFile, validateCoverageReport} from './utils'
+import {detectFormat, validateCoverageReport} from './utils'
 
 const TRACE_ID_HTTP_HEADER = 'x-datadog-trace-id'
 const PARENT_ID_HTTP_HEADER = 'x-datadog-parent-id'
 const errorCodesStopUpload = [400, 403]
 
 const MAX_REPORTS_PER_REQUEST = 10
 
+const IS_COVERAGE_REPORT = (file: string): boolean => {
+  if (path.extname(file) !== '.xml') {
+    return false
+  }
+
+  const filename = path.basename(file)
+
+  return (
+    filename.startsWith('jacoco') || filename.includes('Jacoco') // jacoco*.xml, *Jacoco*.xml
```

Comment:
is this case sensitive? If `jacoco` is not at the start of the filename, does it have to be capitalized? I wonder if we could simply do 
```suggestion
    filename.toLowerCase().includes('jacoco')
```
this wouldn't work if we _don't_ want to include filenames of the type `thing-jacoco-thing.xml` but we do want to include `thing-Jacoco-thing.xml`

Reply:
yup, that was case-sensitive and intentional